app/javascript/components/common/CopyToClipboardButton.tsx

```diff
@@ -10,10 +10,6 @@ const KEY_NAMES_LEGACY = Object.freeze({
 })
 
 export function CopyToClipboardButton({ textToCopy }: { textToCopy: string }) {
-  if (window.navigator.clipboard === undefined) {
```

Comment:
Is this because we will only support modern browsers?

Reply:
Yes, but also because it doesn't work on localhost without this, which can introduce bugs. If we can guard this so it's there for production, that'd be good. But we need to fallback to something else instead, rather than just leave it blank.

I think the real solution is to render the widget regardless, then have this guard on the functionality that uses it, not on the whole component.